src/interface/hooks/use-post-present-proof-send-request.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * This function is used to send with POST a proposal for a present proof
+ */
+import { useCallback, useState } from 'react'
+import post from '../api/helpers/post'
+
+export default function usePostPresentProofSendRequest() {
+  const path = '/present-proof-2.0/send-request'
+
+  const [error, setError] = useState(null)
+  const [status, setStatus] = useState('idle')
+
+  const createBody = (comment, connectionId, proposal, validity) => {
+    const reqPrs4zkProofs = [
+      {
+        name: 'expiration_dateint',
+        p_type: '>=',
+        p_value: validity.split('-').join('') * 1,
```

Comment:
this looks a like a good place for bug to happen when this is a srtring and we perform some string methods and later treating like a number. If JS magic for some reason would fail or would not work, this could turn into a bug.

Reply:
Agreed this is ugly. Will find a better way